scripts/legacy_expression_tool/README.md

```diff
@@ -235,7 +246,4 @@ change the`expressionLanguage` field in the document. This means that the legacy
 `expressionLanguage` field will still be set to the default value or legacy. This would make expression evaluation to
 fail, propagating the value of the attribute as the expression literal to the context broker.
 
-In order to avoid this, it is recommended to set `expressionlanguage` to `jexl`, so the script will set the
-`expressionLanguage` field to `jexl` in the documents where the legacy expressions are replaced. This will ensure that  
-modified documents include the `expressionLanguage` field set to `jexl` and the expressions are evaluated correctly.
-This is specially important when replacing expressions at device level.
+In sum, it is recommented use always the `--expressionlangauge` parameter (note that is't optional) and doing it with a value different from `ignore`.
```

Comment:
It is better to suggest `jexl` value for `--expresionlanguage`. As an example, using `delete` option would not help.

```suggestion
To avoid this, it is recommented use always the `--expressionlangauge` parameter set to `jexl` (note that is't optional) and doing it with a value different from `ignore`.
```